core/arch/arm/plat-ls/conf.mk

```diff
@@ -103,12 +103,14 @@ CFG_TZDRAM_SIZE ?= ( CFG_TEE_OS_DRAM0_SIZE - CFG_SHMEM_SIZE)
 #CFG_SHMEM_START (Non-Secure shared memory) needs to be 2MB aligned boundary for TZASC 380 configuration.
 CFG_SHMEM_START ?= ((CFG_DRAM0_BASE + CFG_DRAM0_SIZE) - CFG_SHMEM_SIZE)
 $(call force,CFG_ARM64_core,y)
+CFG_USER_TA_TARGETS ?= ta_arm64
 else
 #In ARMv7 platform CFG_SHMEM_SIZE is different to that of ARMv8 platforms.
 CFG_TZDRAM_START ?= ((CFG_DRAM0_BASE + CFG_DRAM0_SIZE) - CFG_TEE_OS_DRAM0_SIZE)
 CFG_TZDRAM_SIZE ?= ( CFG_TEE_OS_DRAM0_SIZE - (2*CFG_SHMEM_SIZE))
 #CFG_SHMEM_START (Non-Secure shared memory) needs to be 2MB aligned boundary for TZASC 380 configuration.
 CFG_SHMEM_START ?= ((CFG_DRAM0_BASE + CFG_DRAM0_SIZE) - (2*CFG_SHMEM_SIZE))
+CFG_USER_TA_TARGETS ?= ta_arm32
```

Comment:
Is this needed? This part is for 32-bit TEE core, `ta_arm32` should be the default for `CFG_USER_TA_TARGETS`.

Reply:
> Is this needed? This part is for 32-bit TEE core, `ta_arm32` should be the default for `CFG_USER_TA_TARGETS`.

I have removed the line from else part.

> Now, it's your call to choose not to build 32-bit TA libs by default, but it's not a fix?

For NXP build environment, CROSS_COMPILE is set to 32-bit or 64-bit toolchain depending on the platform.

To accommodate the  release upgrade of the OPTEE-OS , in the existing NXP build frame-work, this change is important.

Thanks for your review and replies.
Raising the new pull request with above changes.